src/services/teams/hooks/collection.js

```diff
@@ -1,15 +1,14 @@
 const { BadRequest } = require('@feathersjs/errors');
 const { ObjectId } = require('mongoose').Types;
 
-
 /**
  * If Array use it.
  * If Object use Object.keys.
  * Otherwise create array with one element.
  * @private
  * @collection
  */
-const mapToArray = (e) => (isArray(e) ? e : (isObject(e) ? Object.values(e) : [e]));
+const mapToArray = (e) => (isArray(e) ? e : isObject(e) ? Object.values(e) : [e]);
```

Comment:
Codacy found an issue: [Do not nest ternary expressions.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6074601)